src/libs/feature-previews-config.ts

```diff
@@ -150,6 +151,15 @@ const featurePreviewsConfig: readonly FeaturePreview[] = [
     )}`,
     lastUpdated: '12/12/2024',
   },
+  {
+    id: GCP_BATCH,
+    title: 'Run workflows on GCP Batch',
+    description:
+      'Enabling this feature allows workflows to be processed by the GCP Batch service, rather than the former Cloud Life Sciences API.',
+    groups: ['preview-batch'],
```

Comment:
It seems that I can turn on this feature without being in the `preview-batch` group.


https://github.com/user-attachments/assets/15a92e1d-273c-45c7-843b-aebc40ca02ae

Reply:
yeah all feature-previews are visible in every environment except prod
https://github.com/DataBiosphere/terra-ui/blob/f2e39d644cfa2e68b5eab8eb349d9e365b61c9ed/src/libs/feature-previews.js#L30

so this is expected behavoir